packages/nimble-components/src/radio-group/tests/radio-group.stories.ts

```diff
@@ -75,7 +75,7 @@ const metadata: Meta<RadioGroupArgs> = {
         },
         name: {
             description:
-                'Radio buttons whose values are mutually exclusive should set the same `name` attribute. Setting the name on the group sets it on all child radio buttons.'
+                'Radio buttons whose values are mutually exclusive should set the same `name` attribute. Setting the name on the group sets it on all child radio buttons. When using radio buttons in an Angular form, you must explicitly set either `name` or `formControlName` on each radio button rather than only setting `name` on the group.'
```

Comment:
Could you update the Angular example app to follow this guidance?

Reply:
Also not 100% clear on the guidance, can't tell if it's saying Angular needs to set the same name on the group and all the children or just the children. I think I would suspect just the children.

Reply:
I've updated the example and changed the guidance to explicitly say that setting the name on the group doesn't work in the Angular forms case.